src/services/storage-service.spec.js

```diff
@@ -568,11 +568,11 @@ describe('storage-service', () => {
 
       describe('and the user is room collaborator and the room owner does not have a storage plan', () => {
         beforeEach(async () => {
-          const collaboratorUser = await setupTestUser(container, {
+          const collaboratorUser = await createTestUser(container, {
```

Comment:
Thanks for this, I noticed the naming discrepancy a few times but never streamlined!

Reply:
I had to change the function code anyways, so I thought, it's time :smile: